docs/baseline-protocol-code/branch-naming.md

```diff
@@ -0,0 +1,23 @@
+# Branch Naming Standard
+
+## Main
+
+*main* - main development branch, feature and release branches branched from it, changes only through the PR process.
```

Comment:
is there a release branch
maybe mention bugfix branches branched from it

Reply:
Implemented